os/include/pthread.h

```diff
@@ -204,6 +204,12 @@
 #define _PTHREAD_MFLAGS_INCONSISTENT  (1 << 1)	/* Mutex is in an inconsistent state */
 #define _PTHREAD_MFLAGS_NRECOVERABLE  (1 << 2)	/* Inconsistent mutex has been unlocked */
 
+/*
+ * Maximum values of pthread key operation
+ */
+#define PTHREAD_KEYS_MAX              CONFIG_NPTHREAD_KEYS
+#define PTHREAD_DESTRUCTOR_ITERATIONS CONFIG_NPTHREAD_DESTRUCTOR_ITERATIONS
```

Comment:
Is this name should be changed to PTHREAD_DESTRUCTOR_ITERATIONS_MAX ? just like above PTHREAD_KEYS_MAX ?

Reply:
PTHREAD KEY MAX and PTHREAD DESTRUCTOR ITERATIONS are written in POSIX. I just got them from POSIX. You can find them at my previous comment.